src/content/whats-new/whats-new.md

```diff
@@ -1,3 +1,19 @@
+**May 11, 2018**
+
+## Utilities
+
+There is a new section in the left navigation called Utilities which gives guidance on common pattern handlings.
+Unlike our other sections, these guidelines offer usage guidance with minimal code documentation. To read more about each of these utilities see the sections below:
```

Comment:
"Unlike ~~our other sections~~ **the component** section," 

There are several other sections that also don't offer much code documentation like Style, Getting Started and Resources.